src/object/interrupt.c

```diff
@@ -142,7 +142,7 @@ void invokeIRQHandler_AckIRQ(irq_t irq)
         return;
```

Comment:
The case where an IRQ is acked on a different core than the one it was delivered on would also need a special update for deactivating.

Reply:
yep, For PPIs only.

For SPIs, the IRQ can be acked by a different core.